.gitignore

```diff
@@ -1,18 +1,31 @@
+# Project
 /func
 /func_*
+/cmd/func.yaml
 /templates/go/cloudevents/go.sum
 /templates/go/http/go.sum
 /templates/typescript/cloudevents/build
 /templates/typescript/http/build
 /coverage.out
 /bin
 /target
-.DS_Store
+
+# Nodejs
 node_modules
+
+# Python
 __pycache__
-/coverage.out
-/bin
 
+# VSCode
 .vscode
+
+# IntelliJ
 .idea
+
+# Operating system temporary files
 .DS_Store
+
+# Non relevant vendor files
+BUILD.bazel
+OWNERS
+OWNERS_ALIASES
```

Comment:
Although these additions are useful, I would prefer this to be in a separate PR as they are unrelated to the feature you are adding. Not a deal breaker, but that is my preference.

Reply:
as requested: https://github.com/knative-sandbox/kn-plugin-func/pull/1209